### PR TITLE
Added possibility for selecting a specific adr algorithm

### PR DESCRIPTION
**Pull request by MStenkaer**
Added dropdown menu for selecting a specific adr algorithm when creating a device profile.

Depends on https://github.com/OS2iot/OS2IoT-backend/pull/192
Needed for resolving https://github.com/OS2iot/OS2IoT-frontend/issues/103

### CODE REVIEW

**Comment by ramogens** [comment]
Thank you very much for making this PR, @MStenkaer. I'll start the process of getting a review, but I'm afraid I can't tell when it will be made as we have no new releases planned right now.

**Comment by ramogens** [comment]
A quick update about expected time for review... as of now it will be in October/November with a following release if it clears the review.
/Mogens, product coordinator

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,18762 @@
 {
   "name": "os2iot-frontend",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (AramAlsabti) [inline package-lock.json:4]: Reset this as no dependency changes were made

**Review by AramAlsabti — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (MStenkaer) [reply package-lock.json]: Thank you for the comment. The file has been reverted.

**Review by AramAlsabti — APPROVED** [review]
Three approved PRs in one go. Not bad!

**Pull request merged**